### PR TITLE
Geth 1.4.8 "DAO Wars"

### PR DESCRIPTION
**Pull request by karalabe**

### CODE REVIEW

**Comment by robotally** [comment]
:exclamation: Pull request against `master` :exclamation:

| Vote | Count | Reviewers |
| :---: | :---: | :---: |
| :+1: | 0 |  |
| :-1: | 0 |  |

_Updated: Fri Jun 24 14:35:35 UTC 2016_

**Comment by GitCop** [comment]
Thank you for your contribution! Your commits seem to not adhere to the repository coding standards

 * Commit: a9c94cbf48fefe39104ee3495df139ff374dd219
  * Commit subjects should be kept under 80 characters

 * Commit: f31a3a251afad1d4c7169bc530fe60471df36122
  * Commit subjects should be kept under 80 characters

 * Commit: b170a80cdcb380765c711017588b2d148eb1601e
  * Commit subjects should be kept under 80 characters

Please check the [contribution guidelines](https://github.com/ethereum/go-ethereum#contribution) for more details.

---
This message was auto-generated by https://gitcop.com

File: cmd/utils/flags.go [hunk 1]
```diff
@@ -181,6 +181,10 @@ var (
 		Usage: "Target gas limit sets the artificial target gas floor for the blocks to mine",
 		Value: params.GenesisGasLimit.String(),
 	}
+	DAOSoftForkFlag = cli.BoolFlag{
+		Name:  "dao-soft-fork",
+		Usage: "Vote for the DAO soft-fork, temporarilly decreasing the gas limits",
```
Comment (thijstriemstra) [inline cmd/utils/flags.go]: typo: temporarily

File: core/dao_test.go [hunk 2]
```diff
@@ -0,0 +1,358 @@
...
+//     if (msg.value > 0) {
+//       _dao.call.value(msg.value)();
+//     } else {
+//       _dao.call(bytes4(sha3("refund()")));
+//     }
+//   }
+// }
+var proxyDeployCode = common.FromHex("606060405260405160208060dd83395060806040525160008054600160a060020a031916821790555060a98060346000396000f360606040523615600a575b60486000341115604a576000805473ffffffffffffffffffffffffffffffffffffffff1690349060609081818185876185025a03f1925050505060a7565b005b600080547f590e1ae300000000000000000000000000000000000000000000000000000000606090815273ffffffffffffffffffffffffffffffffffffffff9091169163590e1ae3916064919060048183876161da5a03f1505050505b56")
+
+// Tests that unannimous votes are handled correctly (i.e. the simple case).
```
Comment (thijstriemstra) [inline core/dao_test.go]: typo: unanimous

**Pull request merged**